src/ButtonAnchor.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+export default class ButtonAnchor extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+  render() {
+    let { href, role, tabIndex, disabled, style } = this.props;
+
+    if (disabled) {
+      tabIndex = -1;
+      style = { pointerEvents: 'none', ...style };
+    }
+
+    return (
+      <a
+        {...this.props}
+        style={style}
+        disabled={null}
+        // we want to make sure there is a href attribute on the node
+        // otherwise, the cursor incorrectly styled (except with role='button')
+        href={href || ''}
+        role={role || 'button'}
+        // http://getbootstrap.com/javascript/#dismiss-on-next-click
+        tabIndex={tabIndex || 0}
+        onClick={this.handleClick}
+      />
+    );
+  }
+
+  handleClick(event) {
+    let { disabled, onClick } = this.props;
+
+    event.preventDefault();
```

Comment:
I'm clearly a bit confused here but I think that's what we want. If a button has a non-trivial `href`, then per http://getbootstrap.com/css/#callout-buttons-anchor-accessibility it should just be a link with button styling and not otherwise get `role="button"` or whatever.